### PR TITLE
R90 initial draft

### PR DESCRIPTION
**Pull request by chuwy**
WIP

### CODE REVIEW

**Comment by BenFradet** [comment]
I need to add a section regarding the spark config can you ping me when I'll be able to do so @chuwy ?

**Comment by chuwy** [comment]
Hey @BenFradet, you already can add it in `2. Other changes` section. Also, please review `Upgrading` section regarding your changes, so we don't miss anything.

File: _posts/2017-07-09-snowplow-r90-lascaux-released.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+       spark:
+         maximizeResourceAllocation: "true"
+       spark-defaults:
+         spark.executor.instances: "17"
+         spark.yarn.executor.memoryOverhead: "4096"
+         spark.executor.memory: "35G"
+         spark.yarn.driver.memoryOverhead: "4096"
+        // etc
+{% endhighlight %}
+
```
Comment (alexanderdean) [inline _posts/2017-07-09-snowplow-r90-lascaux-released.md]: @BenFradet please add a little context on how this is useful - basically Spark needs more tuning/handholding than EMR, and we are now giving you the tools to do it; plus expect some guides on our Discourse from us and the wider community on how to make use of these custom config options, given your specific event workloads, in the future...

**Comment by alexanderdean** [comment]
Thanks for starting this @chuwy - this was plenty for me to start QA. Finishing off this blog post is lower priority than your other tasks...

File: _posts/2017-07-09-snowplow-r90-lascaux-released.md (hunk 1 above)
Comment (BenFradet) [inline _posts/2017-07-09-snowplow-r90-lascaux-released.md]: good point, added some context

**Comment by alexanderdean** [comment]
Hey @chuwy - some more things to add ASAP next week:

1. The improved rationale section, per my notes in HipChat
2. The fact that the archive_enriched step has moved from Sluice to S3DistCp. This is a huge win in terms of stability and performance
3. The fact that this release currently means that it is not possible to load Redshift via an SSH tunnel, or similar non-standard setups
4. The fact that the Redshift credentials are currently passed into the job step and are visible in the EMR UI. We always recommend that the Redshift user we use to load Redshift has as few credentials as possible

On 3 and 4, we are exploring improvements in the milestone https://github.com/snowplow/snowplow/milestone/143

**Comment by chuwy** [comment]
Hey @alexanderdean this is ready for review.

**Pull request merged**